src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/_event.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
```

Comment:
Is this an old file?

Reply:
This looks like an old file, I'll delete.